make/harbor.cfg

```diff
@@ -109,22 +106,31 @@ token_expiration = 30
 #Set to "adminonly" so that only admin user can create project.
 project_creation_restriction = everyone
 
-#The follow configurations are for Harbor HA mode only
-#####################################################
-#the address of the mysql database.
+#######Harbor DB configuration section#######
+#The address of the mysql database. Only need to change when using external db.
 db_host = mysql
 
+#The password for the root user of mysql db, change this before any production use.
+db_password = root123
+
 #The port of mysql database host
 db_port = 3306
 
 #The user name of mysql database
 db_user = root
-#The redis server address
+
+##### End of Harbor DB configuration#######
```

Comment:
better to add a blank line after this section ending.
If ' # End of' then add a space after configuration. Same for the #######Harbor DB configuration section#######. Maybe call it "####### Beginning of ..." to align with ' End of ..'